app/javascript/app/document-capture/components/button.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button({ type, onClick, children, isPrimary, isDisabled, className }) {
+  const classes = ['btn', isPrimary && 'btn-primary btn-wide', className].filter(Boolean).join(' ');
+
+  // Disable reason: We can assume `type` is provided as valid, or the default `button`.
```

Comment:
QQ about these "disable reason" lines -- do we want them as close as possible to the actual `eslint-disable` lines?

Reply:
> QQ about these "disable reason" lines -- do we want them as close as possible to the actual `eslint-disable` lines?

In my mind, they should be, yes. I could probably move this inside the parentheses at least.

Now that you mention it, it'd been on my mind that one of the new features of ESLint 7.0 is [support for comment descriptions](https://eslint.org/docs/user-guide/migrating-to-7.0.0#descriptions-in-directive-comments), which is very helpful for this exact usage. I'd originally considered us blocked from this since CodeClimate only supports up to ESLint 6.x, but since we're no longer using CodeClimate for ESLint, we could probably upgrade this now.

Reply:
> I could probably move this inside the parentheses at least.

Updated in 2c8ac14.

I'll plan to follow-up with ESLint upgrade in a separate pull request.